installer/install.iss

```diff
@@ -1048,14 +1048,27 @@ end;
 
```

Comment:
The commit message is a bit cryptic because it is so short. Also, `build-extra` is about more than just the installer, therefore I would ask to prefix the commit message with the "area code" `installer:`. Something like this:

```
installer: make symlink support detection more robust

When running the installer from an administrator account, currently the
detection fails whether regular users can create symbolic links.

Fix this by detecting whether running in Windows 10 Creators Update or
later in Developer Mode, falling back to the previous method.
```